h2o-docs/src/booklets/v2_2015/source/GLMBooklet.tex

```diff
@@ -1398,6 +1399,128 @@ \subsection{Model Statistics}
+
+where
+
+\begin{itemize}
+\item $\mu_i$ is the prediction
+\item $y_i$ is the real value of the target variable
+\item $k = \frac{1}{\phi}$ is the dispersion parameter estimation
+\end{itemize}
+
+\emph{Note:} You must estimate the dispersion parameter. During initialization, \texttt{compute\_p\_values} and \texttt{remove\_collinear\_columns)} must be set to \texttt{True} to facilitate this estimation process. Also, \texttt{dispersion\_parameter\_method} must be set to \texttt{"ml"}. Since texttt{dispersion\_parameter\_method} only works without regularization, you disable regularization by setting \texttt{lambda} equal to an array of \texttt{0.0}.
+
+\textbf{Gamma:}
+
+$$l(\mu_i (\beta); y_i, w_i) = \frac{w_i}{\phi} \log \big( \frac{w_i y_i}{\phi \mu_i} \big) - \frac{w_i y_i}{\phi \mu_i} - \log (y_i) - \log \big(\Gamma \big(\frac{w)i}{\phi} \big) \big)$$
+
+where
+
+\begin{itemize}
+\item $\mu_i$ is the prediction
+\item $y_i$ is the real value of the target variable
+\item $\phi$ is the dispersion parameter estimation
+\end{itemize}
+
+\emph{Note:} You must estimate the dispersion parameter. During initialization, \texttt{compute\_p\_values} and \texttt{remove\_collinear\_columns)} must be set to \texttt{True} to facilitate this estimation process. Also, \texttt{dispersion\_parameter\_method} must be set to \texttt{"ml"}. Since texttt{dispersion\_parameter\_method} only works without regularization, you disable regularization by setting \texttt{lambda} equal to an array of \texttt{0.0}.
+
+\textbf{Multinomial:}
+
+$$l(\mu_i(\beta); y_i) = w_i \log (\mu_i)$$
+
+where $\mu_i$ is the predicted probability of the actual class $y_i$
+
+\textbf{Tweedie:}
+
+The Tweedie calculation is located in the section \ref{par:tweevar}.
+
+\emph{Note:} You must estimate the dispersion parameter. During initialization, \texttt{compute\_p\_values} and \texttt{remove\_collinear\_columns)} must be set to \texttt{True} to facilitate this estimation process. Also, \texttt{dispersion\_parameter\_method} must be set to \texttt{"ml"}. Since texttt{dispersion\_parameter\_method} only works without regularization, you disable regularization by setting \texttt{lambda} equal to an array of \texttt{0.0}.
+
+\paragraph{Final AIC Calculation}
+
+The final AIC in the output metric is calculated using the standard formula, utilizing the previously computed log likelihood.
+
+$$\text{AIC} = -2LL + 2p$$
+
+where
+
+\begin{itemize}
+\item $p$ is the number of parameters estimated in the model
+\item $LL$ is the log likelihood
+\end{itemize}
+
+To manage computational intensity, \texttt{calc\_like} is used. This parameter was previously only used for HGLM models, but its utilization has been expanded. By default, \texttt{calc\_like=False}, but you can set it to \texttt{True} to enable the calculation of the full log likelihood and full AIC. This computation is performed during the final scoring phase after the model finishes building.
```

Comment:
Please change

By default, \texttt{calc\_like=False}, but you can set it to \texttt{True} to enable the calculation of the full log likelihood and full AIC.

to

By default, \texttt{calc\_like=False}, but you can set it to \texttt{True} and HGLM to False to enable the calculation of the full log likelihood and full AIC.